buildkite/schedule_and_publish/publish_benchmark_alerts_on_pull_requests.py

```diff
@@ -0,0 +1,26 @@
+from models.notification import Notification
+
+
+def publish_benchmark_alerts_on_pull_requests():
+    notifications = (
+        Notification.query()
+        .filter(
+            Notification.finished_at.is_(None),
+            Notification.type == "pull_comment_alert",
+        )
+        .all()
+    )
+
+    for notification in notifications:
+        if not notification.benchmarkable.baseline:
+            continue
+
+        if not notification.all_runs_with_publishable_benchmark_results_finished():
+            continue
+
+        if notification.benchmarkable.has_high_level_of_regressions():
+            notification.create_pull_comment(
+                "Benchmarks have high level of regressions"
+            )
```

Comment:
Will this end up having the links to the runs that have high levels of regressions added later?

Reply:
No links but I can add them.